docs/source/author/writing_guide.rst

```diff
@@ -0,0 +1,451 @@
+
+.. code-block:: none
+
+  Unordered list:
+  - Item 1
+  - Item 2
+  - Item 3
+
+  Ordered list:
+  1. Item 1
+  2. Item 2
+  3. Item 3
+
+Unordered list:
+
+- Item 1
+- Item 2
+- Item 3
+
+Ordered list:
+
+1. Item 1
+2. Item 2
+3. Item 3
+
+Nested lists can be created by indenting each level by 2 spaces.
+
+.. code-block:: none
+
+  1. Item 1
+    1. A corollary to the above item, indented by 2 spaces.
+    2. Yet another point to consider.
+  2. Item 2
+    * A corollary that does not need to be ordered.
+      * This is indented four spaces, because it's two for each level.
+      * You might want to consider making a new list by now.
+  3. Item 3
+
+1. Item 1
+
+  1. A corollary to the above item, indented by 2 spaces.
+  2. Yet another point to consider.
+
+2. Item 2
+
+  * A corollary that does not need to be ordered.
+
+    * This is indented four spaces, because it's two for each level.
+    * You might want to consider making a new list by now.
+
+3. Item 3
+
+------------------------------------------------------------------------------
+
+Math
+==============================================================================
+
+To include math (either inline or as a block) use the following syntax while
+using LaTeX syntax.
+
+.. code-block:: none
+
+  This is inline math: $ 2 + 2 = 4 $
+
+  This is block math:
+
+  $$ \begin{bmatrix} s & 0 \\ 0 & s \\ \end{bmatrix} $$
+
+Math equations are rendered in MathJax using the LaTeX syntax.
+
+------------------------------------------------------------------------------
+
+Panel (Kordac feature)
+==============================================================================
+
+`Click here to read the documentation on how to create a panel`_.
+
+------------------------------------------------------------------------------
+
+Scratch (Kordac feature)
+==============================================================================
+
+`Click here to read the documentation on how to include an image of Scratch block`_.
+
+------------------------------------------------------------------------------
+
+Table of Contents (Kordac feature)
+==============================================================================
+
+`Click here to read the documentation on how to include a table of contents`_.
+
+------------------------------------------------------------------------------
+
+Tables
+==============================================================================
+
+Tables can be created using the following syntax:
+
+.. code-block:: none
+
+  Colons can be used to align columns.
+
+  | Tables        | Are           | Cool  |
+  | ------------- |:-------------:| -----:|
+  | col 3 is      | right-aligned | $1600 |
+  | col 2 is      | centered      |   $12 |
+  | zebra stripes | are neat      |    $1 |
+
+.. raw:: html
+  :file: ../_static/html_snippets/markdown_example_table.html
+
+The outer pipes (|) are optional, and you don't need to make the raw Markdown
```

Comment:
<img width="809" alt="screen shot 2017-03-16 at 9 05 42 pm" src="https://cloud.githubusercontent.com/assets/6937219/23987067/5d3d20d4-0a8c-11e7-9eb9-586311fa1580.png">

Looks fine to me?